include/deal.II/base/symmetric_tensor.h

```diff
@@ -3546,6 +3546,67 @@ DEAL_II_HOST constexpr inline DEAL_II_ALWAYS_INLINE
 
 
 
+/**
+ * Return the symmetrized version of a full rank-4 tensor, i.e.
+ * as a symmetric rank-4 tensor. This is the version for general dimensions.
+ * @param major_symmetry  This argument decides the presence of major symmetry.
```

Comment:
Since the `enum` has now disappeared, you no longer document what "major" and "minor" symmetry means. Would you mind re-adding that here?